Haulage/DatabaseSetup.cs

```diff
@@ -0,0 +1,62 @@
+using System.Data.SQLite;
+using System.IO;
+
+public static class DatabaseSetup
+{
+    private static string GetDatabasePath()
+    {
+        // Get the base directory of the application
+        string baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+
+        // Combine the base directory with the relative path to the database
+        string relativePath = Path.Combine(baseDirectory, @"..\..\..\..\..\Database\Haulage.db");
```

Comment:
- This should probably have some sort of validation to cover the case when haulage.db can't be found

Reply:
I have now added a Unit Case for this issue, in testing.